GenderPayGap.Extensions.AspNetCore/HttpSession.cs

```diff
@@ -91,8 +91,6 @@ public T Get<T>(string key)
                 return default;
             }
 
-            bytes = Encryption.Decompress(bytes);

```

Comment:
Do we know what the compression was doing? Are we going to need it going forward?

Reply:
I think it was just reducing the size of the session data (which I don't think is important, given that we don't store much in the session and we don't have that many users)